finances/templates/finances/js/calculate_users_price.html

```diff
@@ -13,7 +11,7 @@
   };
 
   function set_prices () {
-    var total_price = get_total_price();
+    var total_price = Number({{price}});
```

Comment:
If the price can be Decimal, use `{{ price I unlocalize }}` (module l18n). Thus, `1,18` will always show `1.18` (it's not always the case due to localization), needed by JS.

Reply:
Didn't know that, thanks !

Reply:
Fix: 8559ae2